zmsadmin/config.example.php

```
@@ -10,8 +10,8 @@
 
 class App extends \BO\Zmsadmin\Application
 {
-    const IDENTIFIER = ZMS_IDENTIFIER;
-    const DEBUG = false;
+    const string IDENTIFIER = ZMS_IDENTIFIER;
```

PHPStan (2.2.7)

[error] Constant App::IDENTIFIER (string) does not accept value string|false.


(classConstant.value)